gato/github/api.py

```diff
@@ -6,11 +6,14 @@
 import zipfile
 import re
 import io
+import json #Jimmy
```

Comment:
```suggestion
import json
```

Remove comment and import if just used for debugging.